docs/source/AdministratorGuide/Systems/ResourceStatus/install.rst

```diff
@@ -112,13 +112,6 @@ If is there any SE to be modified, you can do it as follows::
 
 .. _activateRSS:
```

Comment:
Drop this as well?

And RSS is active by default?

Reply:
Yes, RSS is active by default.